src/modules/onboarding.ts

```diff
@@ -84,7 +84,7 @@ export default class Onboarding {
    * @param signingMethod Method to use for signing
    * @returns Signature used to derive your STARK key pairs
    */
-  protected async signStarkKeyDerivationMessage(
+  protected signStarkKeyDerivationMessage(
```

Comment:
nit: I think this should be async still since it returns a promise? If it complains because there's no await, then slightly prefer making it `async` and keeping the old way of awaiting then returning the result

Reply:
I can add the async back. I don't think it complains about not having an await. I will add the async but just return a promise.